docker-toolchain/.cargo/config

```diff
@@ -0,0 +1,2 @@
+[target.armv7-unknown-linux-gnueabihf]
+linker = "arm-linux-gnueabihf-gcc"
```

Comment:
Not sure if the default `rustflags` imply this already or if this even makes a difference in terms of runtime performance however I have seen this added for binaries targeting `imx6slevk`:
```
rustflags = [
   "-C", "target-feature=+v7,+vfp3,+a9,+neon"
]
```

Reply:
You're welcome to make changes as necessary, as long as you test them. I don't know if that's in the scope of this PR, though, unless those flags are already being used in the existing toolchain. Still, it sounds like something to look into.

Reply:
If you're already using them, and it works, it would make sense to me to either include them in this PR or add them shortly after. If you'd prefer, I could also add them from my end.

Updating documentation to include them (where they are missing) may also help.